diff --git a/test/e2e-tests.sh b/test/e2e-tests.sh
index a200387d9..7338cd2a3 100755
--- a/test/e2e-tests.sh
+++ b/test/e2e-tests.sh
@@ -31,7 +31,7 @@ source "$(dirname "$0")/e2e-common.sh"
 # Script entry point.
 
 initialize "$@"
-
+export CGO_ENABLED=1
 echo "Running E2E tests for: Multi Tenant Channel Based Broker, Channel (v1), InMemoryChannel (v1) , ApiServerSource (v1), ContainerSource (v1) and PingSource (v1beta2)"
 go_test_e2e -timeout=1h -parallel=20 ./test/e2e \
   -channels=messaging.knative.dev/v1:Channel,messaging.knative.dev/v1:InMemoryChannel \
